app/book.py

```diff
@@ -0,0 +1,34 @@
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
+        if display_type == "console":
+            self.display_console()
+        elif display_type == "reverse":
+            self.display_reverse()
+        else:
+            raise ValueError(f"Invalid display type: {display_type}")
+
+    def display_console(self) -> None:
+        print(self.content)
+
+    def display_reverse(self) -> None:
+        print(self.content[::-1])
+
+    def print_book(self, print_type: str) -> None:
+        if print_type == "console":
+            self._print_console_book()
+        elif print_type == "reverse":
+            self._print_reverse_book()
+        else:
+            raise ValueError(f"Invalid print type: {print_type}")
+
+    def _print_console_book(self) -> None:
+        print(f"Printing the book: {self.title}...")
+        self.display_console()
+
+    def _print_reverse_book(self) -> None:
+        print(f"Printing the book in reverse: {self.title}...")
+        self.display_reverse()
```

Comment:
Your code violates Single-responsibility principle. You should create separate class for console and reverse dispaly